src/main/java/com/toptrumps/db/PersistentDataDAO.java

```diff
@@ -73,22 +73,34 @@ public static void updatePersistentData(){
       }
     }
 
-    public static ResultSet getStatistics(){
+    public static Statistics getStatistics(){
 
         Connection conn = ConnectionFactory.getConnection();
         PreparedStatement getStats = null;
         String getStatsSQL = "SELECT * FROM persistent_game_data;";
         ResultSet results = null;
+        Statistics statistics = null;
 
         try{
 
             getStats = conn.prepareStatement(getStatsSQL);
             results = getStats.executeQuery();
+
+            while(results.next()) {statistics = new Statistics(results);}
```

Comment:
You are creating a Statistics object for each result?

Reply:
Nah, just one. Each "row" of a ResultSet is considered an object, and the .next() method identifies when there is a new row. Since the persistent stats just have one row, it's just writing that to an object. 

It's essentially checking just checking if the table has any rows at all - can't do != null since ResultSets are never null

Reply:
Gotcha!